README.md

```diff
@@ -1,75 +1,267 @@
+
+- arguments: `none`
+- options: `none`
+
+**Example:**
+
+```bash
+cudos test
 ```
-cudo keys fund alice -t 1000000000ucudos
+
+---
+
+
+### `cudos node`
+
+> Base command for interaction with a local node ([`cudos node`](https://github.com/CudoVentures/cudos-node) )
+> Run ```cudo node --help``` for info
+
+### `cudos node start`
+> Starts a local CUDOS node
+
+**Example:**
+
+```bash
+cudos node start
 ```
+
+### `cudos node stop`
+> Stops a local CUDOS node
+
+**Example:**
+
+```bash
+cudos node stop
 ```
-cudo keys ls
-[
-  {
-    name: 'alice',
-    addr: 'cudos1n4mxz8qu8zestvkk790ans2klj4qyzutcl550q',
-    balance: { denom: 'ucudos', amount: '1000000000' }
-  },
-  {
-    name: 'juliette',
-    addr: 'cudos1nd8ppd846rhnuulujv7ay0vrppa5md705tvp8d',
-    balance: { denom: 'ucudos', amount: '100043434300' }
-  }
-]
+
+### `cudos node status`
+> Gives a status of a local CUDOS node - online/offline
+
+**Example:**
+
+```bash
+cudos node stop
 ```
-### compile
+
+### `cudos node keys`
+> List keys of a local CUDOS node
+
+**Example:**
+
+```bash
+cudos node keys
 ```
-cudo compile
+
+---
+
+### `cudos run`
+> Runs a custom javascript file. 
+> Currently in the /scripts directory there are two files: deploy.js and interact.js
+> In the future it should be possibly to supply relevant info by the command line (smart contract name, user adress, etc) as option
+> 
+
+- arguments: `scriptFilePath` `string` `The script file to execute`
+- options: `none`
+  
+
+  ###### How to deploy different smart contracts?
+  > For each of your smart contracts create a separate deploy file ( for example deploy_alpha.js and deploy_beta.js) and run it one by one. Do not forget to edit the contract name in the .js file ( line 4 in the supplied deploy.js file)
+
+**Example:**
+
+```bash
+cudos run scripts/deploy.js
+cudos run scripts/interact.js
+cudos run scripts/yourScriptFile.js
 ```
 
-### starting and stopping cudos-node
+---
+
+### `cudos keys`
+> Manages keystore/accounts
+
+### Subcommands:
+
+#### `cudos keys add accountName`
+> Add account to the keystore
+- arguments: `accountName` `string`
+- options: `none`
+  
+  
+**Example:**
+
+```bash
+cudos keys add accountName
 ```
-cudo node start
+
+#### `cudos keys rm accountName`
+> Removes account from keystore
+- arguments: `accountName` `string`
+- options: `--yes, -y` `No confirmation when deleting a user`
+
+**Example:**
+
+```bash
+cudos keys rm accountName
 ```
+
+#### `cudos keys ls`
+> List all accounts in the keystore
+- arguments: `None`
+- options: `None`
+  
+  
+**Example:**
+
+```bash
+cudos keys ls
 ```
-cudo node stop
+
+#### `cudos keys fund accountName`
+> Fund account with tokens
+- arguments: `accountName` `string`
+- options: 
+  - `--address, -a` `string` `The address of the account`
+  - `--tokens, -t` `string` `The amount of tokens to give` `required`
+  
+  
+**Example:**
+
+```bash
+cudos keys fund accountName --address=sampleAddress --token=10000000ucudos
```

Comment:
We do not need to use --address option if there is accountName. We should use --address only if we want to fund account which is not in the keystore.